package-lock.json

```diff
@@ -82,9 +82,9 @@
             }
         },
         "@types/adm-zip": {
```

Comment:
Merging from `master` branch will probably make most of these changes in `package-lock.json` and `package.json` (for versions) go away; it will get the dependency update from #1889.